Module not found: Can't resolve '@lattice/mui-recharts'

Error ocurring when starting the test-demo-app.

Module not found: Can't resolve '@lattice/mui-recharts' in '/Users/elmasse/Services/lattice/lattice/test-demo-app/src/components/dashboard'

--- a/packages/react-scripts/template/src/components/dashboard/AverageRevenue.js
+++ b/packages/react-scripts/template/src/components/dashboard/AverageRevenue.js
@@ -3,7 +3,7 @@ import React, { Component } from 'react';
 import Typography from '@material-ui/core/Typography';
 
 // Lattice
-import { LineChart, Line, ResponsiveContainer } from '@lattice/mui-recharts';
+import { LineChart, Line, ResponsiveContainer } from '@latticejs/mui-recharts';
 import Widget from '@latticejs/widgets/Widget';
 
 const data = [

